lnrpc/invoicesrpc/invoices.proto

```diff
@@ -82,6 +82,11 @@ message AddHoldInvoiceRequest {
 
     /// Whether this invoice should include routing hints for private channels.
     bool private = 9 [json_name = "private"];
+
+    /**
+    The list of private channel ids that will be used as hop hints.
+    */
+    repeated uint64 private_hops = 10 [json_name = "private_hops"];
```

Comment:
Wondering whether name should be `private_channels`? But we also have hop _hints_...

Reply:
Changed to `private_channels` since I think it's more descriptive